app/models/like.rb

```diff
@@ -1,4 +1,10 @@
 class Like < ApplicationRecord
-  belongs_to :user
-  belongs_to :post
+  belongs_to :author, foreign_key: :author_id, class_name: 'User'
+  belongs_to :post, foreign_key: :post_id, class_name: 'Post'
+
+  after_create :update_likes_counter
+
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
**[OPTIONAL]**

- [ ] Kindly make this method private because this method should not be accessed by the users directly 👍 😊